tests/plugins/tts/WebTTSEngine.test.js

```diff
@@ -32,24 +32,22 @@ describe('WebTTSSound', () => {
             expect(sound.rate).toBe(2);
         });
 
-        test('reloading does not resolve the original promise', () => {
+        test('reloading does not resolve the original promise', async () => {
```

Comment:
really digging the use of async/await here 👏

Reply:
This code is kind of awkward, but it's slightly _less_ awkward because of async/await :P